ggml/src/ggml-cuda/fattn-common.cuh

```diff
@@ -315,14 +315,14 @@ static __device__ __forceinline__ void quantize_q8_1_to_shared(
 
     float vals[sizeof(int)] = {0.0f};
 #pragma unroll
-    for (int l = 0; l < sizeof(int); ++l) {
+    for (size_t l = 0; l < sizeof(int); ++l) {
```

Comment:
```suggestion
    for (int l = 0; l < int(sizeof(int)); ++l) {
```

This is going to be unrolled anyways but 64 bit variables should be used very sparingly in code that is sensitive to register pressure.

Reply:
No problem. I'll update it accordingly.